src/main/cli/cli.c

```diff
@@ -3594,6 +3594,9 @@ static void cliExit(const char *cmdName, char *cmdline)
     cliMode = false;
     // incase a motor was left running during motortest, clear it here
     mixerResetDisarmedMotors();
+    if (strncasecmp(cmdline, "-n", 2) == 0) {
```

Comment:
```suggestion
    if (strcasecmp(cmdline, "-n") == 0) {
```
strncasecmp will accept anything starting with '-n' (-now_reboot_please). `strncasecmp` is used a lot in cli, but correctly only in few places.
`n` is not preventing any overflows in this case (the function does not overwrite anything).

Reply:
This is now sorted.